### PR TITLE
[FEAT] Add Descriptions for Rare and Luxury Keys

### PR DESCRIPTION
**Pull request by eliasSFL**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/lib/i18n/dictionaries/portugueseDictionary.ts [hunk 1]
```diff
@@ -2274,6 +2274,9 @@ const gameDescriptions: Record<GameDescriptions, string> = {
   "description.tulip.bulb": "Um ingresso usado durante a Florada da Primavera",
   "description.prizeTicket": "Um ingresso para entrar nos sorteios de prêmios",
   "description.treasure.key": "Visite o Plaza para desbloquear sua recompensa",
+  "description.rare.key": "Visite a Beach para desbloquear sua recompensa",
```
Comment (Rofrtd) [inline src/lib/i18n/dictionaries/portugueseDictionary.ts:2277]:
```suggestion
  "description.rare.key": "Visite a praia para desbloquear sua recompensa",
```

File: src/lib/i18n/dictionaries/portugueseDictionary.ts [hunk 2]
```diff
@@ -2274,6 +2274,9 @@ const gameDescriptions: Record<GameDescriptions, string> = {
...
+  "description.luxury.key":
+    "Visite a Plaza perto de Woodlands para desbloquear sua recompensa",
```
Comment (Rofrtd) [inline src/lib/i18n/dictionaries/portugueseDictionary.ts:2279]:
```suggestion
    "Visite o Plaza perto de Woodlands para desbloquear sua recompensa",
```

**Review by Rofrtd — COMMENTED** [review]
Just a few changes for the portuguese translation.

**Comment by eliasSFL** [comment]
@Rofrtd I've committed your suggestions

**Review by Rofrtd — APPROVED** [review]

**Pull request merged**